cloud-init/copr_build.py

```diff
@@ -120,9 +120,12 @@ def main(srpm, copr_conf=DEFAULT_COPR_CONF, dev=None):
 
     # 2018-03-05: adding sleep to let builds ramp up
     # this after a {'state': ['Not a valid choice.']} exception
-    time.sleep(180)
+    for timeout in range(1, 10):
+        time.sleep(30)
```

Comment:
Do we have any experience with how quickly tasks show up?  i hate waiting 30 seconds if we could have tasks quicker.

```
factor=6
for timeout in [5] * factor + [10] * factor + [30] * factor:
   time.sleep(timeout)
```
Still close to the 300 seconds but tries quicker sooner and the scales up it's timeout
```
>>> [5] * factor + [10] * factor + [30] * factor
[5, 5, 5, 5, 5, 5, 10, 10, 10, 10, 10, 10, 30, 30, 30, 30, 30, 30]
>>> sum([5] * factor + [10] * factor + [30] * factor)
270
```

Reply:
Sometimes the tasks show up in just a few seconds, sometimes it takes minutes, I guess it depends on the load of the copr build infrastructure. I updated the PR with the solution you suggested.